api/document.go

```diff
@@ -59,7 +60,7 @@ func PostDocumentHandler(res http.ResponseWriter, req *http.Request) {
 	var questions []models.Question
 	for _, coord := range data.Coords {
 		q := models.Question{
-			Document: data.ID,
+			Document: html.EscapeString(data.ID),
```

Comment:
This is just an hash and it is used in the URL. I don't think we need to escape it